api/RTCIceCandidate.json

```diff
@@ -124,6 +124,253 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "candidateInfo_parameter": {
+          "__compat": {
+            "description": "<code>candidateInfo</code> parameter",
+            "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcicecandidateinit",
```

Comment:
> @Elchi3 @sideshowbarker Do you have any recommendations for parameter spec URLs? There aren't a lot of existing examples. This one points to the spec for the type of the parameter (the dictionary), but I've also seen some of these point to the specification saying that this is a parameter of the method. It makes sense to me that the members would get the dict specification URL, but I'm not sure what to choose at this boundary.

I think it depends a lot on how the spec is structured, and what amount of detail in provides for developers (rather than just for implementors), and where it provides that developer-focused information. In other words, I think it’s a category where the answer is a not-so-satisfying *“it depends”* — in practice, it needs looking at case-by-case.

In this case, the https://w3c.github.io/webrtc-pc/#dom-rtcicecandidateinit URL clearly seems like right target to be citing.

Reply:
Thank you, @sideshowbarker. As "it depends" go, that's not a bad one. 👍